### PR TITLE
feat(build): add support for angular/progressive

### PR DESCRIPTION
**Pull request by mgechev**
@jeffbcross @alxhub I'd appreciate your feedback here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ _Does not rely on any global dependencies._
 - [Configuration](#configuration)
 - [How to extend?](#how-to-extend)
 - [Running tests](#running-tests)
+- [Progressive apps](#progressive-apps)
```
Comment (addyosmani) [inline README.md]: s/Progressive apps/Progressive Web Apps

File: README.md [hunk 2]
```diff
@@ -115,6 +116,21 @@ npm run e2e.live
 ```
 You can learn more about [Protractor Interactive Mode here](https://github.com/angular/protractor/blob/master/docs/debugging.md#testing-out-protractor-interactively)
 
+# Progressive apps
+
+`angular2-seed` supports progressive apps with [angular/progressive](https://github.com/angular/progressive).
```
Comment (addyosmani) [inline README.md]: Same comment as earlier. https://github.com/angular/progressive specifically tries to help with building https://developers.google.com/web/progressive-web-apps

Comment (mgechev) [inline README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (mgechev) [inline README.md]: Fixed.

**Comment by jeffbcross** [comment]
This is awesome! @alxhub has been refactoring and adding some features this week so he should comment.

**Comment by mgechev** [comment]
@ludohenin would you review and merge?

**Comment by ludohenin** [comment]
Yep, will do that.

File: package.json [hunk 3]
```diff
@@ -13,6 +13,9 @@
     "build.prod": "gulp build.prod --color",
     "build.test": "gulp build.test --color",
     "build.test.watch": "gulp build.test.watch --color",
+    "generate.manifest.dev": "gulp generate.manifest.dev --color",
+    "generate.manifest.prod": "gulp generate.manifest.prod --color",
+    "docs": "npm run gulp -- build.docs --color && npm run gulp -- serve.docs --color",
```
Comment (ludohenin) [inline package.json]: `docs` can be removed, this has been refactored by @d3viant0ne and is no longer there.

File: gulpfile.ts [hunk 4]
```diff
@@ -19,6 +19,14 @@ gulp.task('build.dev', (done: any) =>
...
               done));
 
+gulp.task('generate.manifest.dev', (done: any) => {
+  runSequence('build.manifest', done);
+});
+
+gulp.task('generate.manifest.prod', (done: any) => {
+  runSequence('build.manifest', done);
+});
+
```
Comment (ludohenin) [inline gulpfile.ts]: Both tasks look to be the same.
I think this can simply be removed.

File: package.json [hunk 5]
```diff
@@ -13,6 +13,9 @@
     "build.prod": "gulp build.prod --color",
     "build.test": "gulp build.test --color",
     "build.test.watch": "gulp build.test.watch --color",
+    "generate.manifest.dev": "gulp generate.manifest.dev --color",
+    "generate.manifest.prod": "gulp generate.manifest.prod --color",
```
Comment (ludohenin) [inline package.json]: Based on the above comment, this can be reduced to `generate.manifest`.

**Comment by mgechev** [comment]
@ludohenin good catches. Fixed.

**Pull request merged**